packages/platforms/rivet/src/actor_driver.ts

```diff
@@ -8,6 +8,10 @@ export class RivetActorDriver implements ActorDriver {
 		this.#ctx = ctx;
 	}
 
+	get context(): unknown {
+		return this.#ctx;
```

Comment:
Relevant to Redis comment: I'm indifferent on if we change this to `return { ctx }`. I don't see a use case where you'd need any, but that's only because I wrote the Rivet runtime.

Reply:
I added it just to be consistent.